docs/notebooks/formulary/mms.ipynb

```diff
@@ -0,0 +1,241 @@
+{
```

Comment:
I do really like the idea of connecting `plasmapy` to some real world scenarios.  Love it! Great idea!